dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -72,6 +72,34 @@ public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true)
         return lambdaEventDAO.findByOrganization(organization, offset, limit, authorizedRepos);
     }
 
+    @GET
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @Path("/{userid}/{organization}")
+    @Operation(operationId = "getUserLambdaEventsByOrganization", description = "Get all of the Lambda Events for the given user and GitHub organization.",
+            security = @SecurityRequirement(name = ResourceConstants.JWT_SECURITY_DEFINITION_NAME))
+    @ApiOperation(value = "See OpenApi for details")
+    public List<LambdaEvent> getUserLambdaEventsByOrganization(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
+           @ApiParam(value = "userid", required = true) @PathParam("userid") long userid,
+           @ApiParam(value = "organization", required = true) @PathParam("organization") String organization,
+           @ApiParam(value = PAGINATION_OFFSET_TEXT) @QueryParam("offset") @DefaultValue("0") String offset,
+           @ApiParam(value = PAGINATION_LIMIT_TEXT, allowableValues = "range[1,100]", defaultValue = PAGINATION_LIMIT) @DefaultValue(PAGINATION_LIMIT) @QueryParam("limit") Integer limit) {
+        final User loggedInUser = userDAO.findById(authUser.getId());
+        if (loggedInUser.getIsAdmin() || loggedInUser.isCurator()) {
+            final List<Token> githubTokens = tokenDAO.findGithubByUserId(userid);
+            if (githubTokens.isEmpty()) {
```

Comment:
It might be better to dump all of the events corresponding to the specified user.  For example, we might want to use the information caused by a stale token or de-authorized organization.